src/renderer/src/contexts/ModalRouter.tsx

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
why?

Reply:
It was a miss code, probably when I was implementing it, I was too lazy to type something and wrote it like this to test. Sorry